shared/teams/main/team-row.stories.tsx

```diff
@@ -1,27 +1,14 @@
 import * as React from 'react'
 import * as Sb from '../../stories/storybook'
-import * as Constants from '../../constants/teams'
-import * as Container from '../../util/container'
 import TeamRow from './team-row'
-
-const ids = ['1', '2', '3']
-const store = Container.produce(Sb.createStoreWithCommon(), draftState => {
-  draftState.teams = {
-    ...draftState.teams,
-    teamMeta: new Map([
-      ['1', Constants.makeTeamMeta({memberCount: 32, teamname: 'keybase_storybook'})],
-      ['2', Constants.makeTeamMeta({isOpen: true, memberCount: 11947, teamname: 'fan_club'})],
-      ['3', Constants.makeTeamMeta({isOpen: false, memberCount: 234, teamname: 'club_penguin'})],
-    ]),
-  }
-})
+import {fakeTeamIDs, store} from '../stories'
 
 const load = () =>
   Sb.storiesOf('Teams', module)
     .addDecorator(story => <Sb.MockStore store={store}>{story()}</Sb.MockStore>)
```

Comment:
@jakob223 I wrote a thing to help with this: `Sb.updateStoreDecorator`

https://github.com/keybase/client/blob/master/shared/stories/storybook.shared.tsx#L107-L110